crates/rust-analyzer/tests/heavy_tests/main.rs

```diff
@@ -682,3 +685,313 @@ pub fn foo(_input: TokenStream) -> TokenStream {
     let value = res.get("contents").unwrap().get("value").unwrap().to_string();
     assert_eq!(value, r#""```rust\nfoo::Bar\n```\n\n```rust\nfn bar()\n```""#)
 }
+
+#[test]
+fn test_document_symbol_with_hierarchy() {
```

Comment:
Let's remove this test -- heavy tests are very slow, and don't really test much, as the end result is determined by the client anyway.

Reply:
> the end result is determined by the client anyway.

The motivator for this PR is that Sublime displays the results in the order received from the server and doesn't do any sorting on the client side.

Reply:
Yeah, that's exactly my point: the property we want to check is that "client sensibly interprets the results", and we can't do that without the client. 

To put it another way, we could have had the similar test before, with reverse order, and it would not have helped us to find out that the feature is broken in sublime.

Reply:
Okay.  I have removed the tests.  :)